### PR TITLE
Make new features directory and move related docs

### PR DESCRIPTION
**Pull request by thompiler**
Closes: https://github.com/airbrake/airbrake-docs/issues/126

We'll need to update any links in our app to point to the docs that moved.

### CODE REVIEW

File: jekyll/_docs/features/deploy-tracking-troubleshooting.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
 ---
 layout: classic-docs
 title: Deploy tracking troubleshooting
-categories: [airbrake-faq]
+categories: [features]
```
Comment (shifi) [inline jekyll/_docs/features/deploy-tracking-troubleshooting.md]: Should this go under `features` section? It's troubleshooting doc, right?

File: jekyll/_docs/features.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+layout: classic-category
+title: Airbrake Features
```
Comment (shifi) [inline jekyll/_docs/features.md]: `Features` -> `features`?

File: jekyll/_docs/features/deploy-tracking-troubleshooting.md (hunk 1 above)
Comment (thompiler) [inline jekyll/_docs/features/deploy-tracking-troubleshooting.md]: Correct, this is about troubleshooting a specific feature (deploy tracking). Do you think we should have a specific `troubleshooting` directory?

File: jekyll/_docs/features.md (hunk 2 above)
Comment (thompiler) [inline jekyll/_docs/features.md]: Sounds good. Thanks 👍

File: jekyll/_docs/features/deploy-tracking-troubleshooting.md (hunk 1 above)
Comment (thompiler) [inline jekyll/_docs/features/deploy-tracking-troubleshooting.md]: Or perhaps keep it in the `FAQ` section? Our `FAQ` section has a bit too many articles IMO.

Comment (shifi) [inline jekyll/_docs/features/deploy-tracking-troubleshooting.md]: I think we should. I personally wound't look under `features` to find `troubleshooting` doc but that's just me.

Comment (shifi) [inline jekyll/_docs/features/deploy-tracking-troubleshooting.md]: `FAQ` also works for me.

Comment (thompiler) [inline jekyll/_docs/features/deploy-tracking-troubleshooting.md]: Mmk sounds good

**Comment by thompiler** [comment]
PTAL

**Pull request merged**